26-register_registration/resource_types/research_data/dataset.feature

```diff
@@ -0,0 +1,108 @@
+		And can be completed at any later stage
+		And answer to the user support request will be visible on "My page"
+
+	@9140
+	Scenario: User adds zero or more geographical data to the dataset
+		Given User confirms to register data legal to publish
+		When the User writes some free text geographical data
+		# Future: Add support for land, region, county and map reference (line, rectangle, point)
+		Then stored it
+
+	@9140
+	Scenario: User adds zero or more use-references to resource published in NVA
+		Given User confirms to register data legal to publish
+		When the user enters some text, a matching list of published results are displayed 
+		And the User may select one result
+		Then store it as depended on this result
+
+	@9140
+	Scenario: User adds zero or more related-references to resource published in NVA
+		Given User confirms to register data legal to publish
+		When the user enters some text, a matching list of published results are displayed 
+		And the User may select one result
+		Then store it as associated to this result
+
+	@9140
+	Scenario: User adds zero or more comply-to-references to a DMP resource published in NVA
+		Given User confirms to register data legal to publish
+		When the user enters some text, a matching list of published DMPs is displayed
+		And the User may select one result
+		Then store it as comply-to to this result
+
+	@9142
+	Scenario: User sees their own published resources in NVA as default matching list
+		Given User adds zero or more use-references to resources published in NVA
+		When the User enters the input field to search for a resource in NVA
+		Then zero to five published results, sorted newest first, are displayed for selection
+		And a list of matches may replace this list 
+
+	@9145
+	Scenario: User confirms publishing the dataset resource
+		When the User have ended the registration wizard 
+		# (pressed "Save and Present" button)
+		And sees the resources' previewed landingpage 
+		Then the User is informed to review registered data
+		And is informed about the risk of publish illegal data
+		And is infomed to contact user support if in doubt
+
+	@9146
+	Scenario: Dataset elements on landingpage
+		When the User sees the landingpage (preview or published)
+		Then the User sees resource type and title on top
+		And then contributors and their affiliations
+		And then a separate line
+		And then publication date, language, publisher and optional DOI
+		And then a separate line
+		And then the abstract
+		And then a separate line
+		And then the keywords
+		And then a separate line
+		And then geographical information
+		And then a separate line
+		And then publications that uses this dataset
+		And then a separate line
+		And then the datasets content (the files)
+		And then a separate line
+		And then the projects assosiated with this dataset
+		And then a separate line
+		And then the DMPs this dataset complay to
+		And then a separate line
+		And then list other related resources
```

Comment:
Denne er kanskje i overkant utssende-fokusert nå. Personlig syns jeg ikke vi trenger å liste opp felter som også vises for alle andre typer heller. De er dekt av andre Scenarios uansett
```suggestion
	Scenario: Use see Landing Page for Dataset
		When User opens Landing Page for a Dataset
		Then they see then geographical information
		And they see then publications that uses this dataset
		And they see then the projects assosiated with this dataset
		And they see then the DMPs this dataset complay to
		And they see then list other related resources
```